app/views/mailers/notify_mailer/feedback_response_email.html.erb

```diff
@@ -7,7 +7,11 @@
 </p>
 
 <p>
-  We appreciate your help as we work to foster a positive and inclusive environment for all!
+  We appreciate your help as we work to foster a positive and inclusive environment for all.
+</p>
+
+<p>
```

Comment:
These changes should have `<% if Page.find_by(slug: "community-moderation") %>` and use `app_url`

Basically same as the logic you see here:

`app/views/mailers/notify_mailer/trusted_role_email.text.erb`